src/routes/boot.ts

```diff
@@ -328,6 +330,47 @@ const getUser = (con: DataSource, userId: string): Promise<User> =>
     ],
   });
 
+const getMarketingCta = async (con: DataSource, userId: string) => {
+  let marketingCta: MarketingCta | null = null;
+  const redisKey = generateStorageKey(
+    StorageTopic.Boot,
+    'marketing_cta',
+    userId,
+  );
+  const rawRedisValue = await getRedisObject(redisKey);
+
+  if (rawRedisValue === 'SLEEPING') {
+    // TODO: remove log or replace with proper logging
+    console.log('sleeping, not fetching from db');
+    return null;
+  } else {
+    marketingCta = JSON.parse(rawRedisValue);
+  }
```

Comment:
My reasoning behind this is to reduce the amount of database queries when the user have no marketing CTAs available

Reply:
Alright, not against it 👍
Maybe just make it a const :)

Reply:
Set `SLEEPING` to be a const. Could also make it a global const that can be re-used too maybe?

Reply:
Yeah sorry global const indeed so we can use it more often.
Maybe could be in redis.ts or something

Reply:
Put it into a enum and called it RedisMagicValues 🪄  Didn't know of of a better name for it. Thought of using a enum so it could possibly be extended in the future